src/main/java/org/apache/mesos/hdfs/util/HDFSConstants.java

```diff
@@ -1,36 +1,39 @@
 package org.apache.mesos.hdfs.util;
 
-public class HDFSConstants {
+/**
+ * defines the constants shared between the scheduler and the executors
+ */
+public interface HDFSConstants {
 
   // Total number of NameNodes
   // Note: We do not currently support more or less than 2 NameNodes
-  public static final Integer TOTAL_NAME_NODES = 2;
+  Integer TOTAL_NAME_NODES = 2;
```

Comment:
the beauty of them being in an Interface... is that they are public static final :)  less code more power.